client/src/test/java/com/influxdb/client/WriteApiTest.java

```diff
@@ -1012,7 +1013,15 @@ void userAgent() throws InterruptedException {
 
         String userAgent = recordedRequest.getHeader("User-Agent");
 
-        Assertions.assertThat(userAgent).startsWith("influxdb-client-java/7.");
+        String currentVersion = UserAgentInterceptor.class.getPackage().getImplementationVersion();
```

Comment:
Added a test change here.  Jakub, does this work for you?  Does it not defeat Utku Aydin's intent for this assertion?